### DISCUSSION TITLE
Docs reorganization for Codeception 5?

### DISCUSSION THREAD

**Discussion by ThomasLandauer** [Codeception/Codeception#6443] [General]
@DavertMik You somewhere mentioned a reorganization of the docs; what are your plans?

I think the duplication between the "Guides" (e.g. https://codeception.com/docs/03-AcceptanceTests) and its corresponding Module (https://codeception.com/docs/modules/WebDriver) is a source of confusion. So I'm suggesting:
1. Strip all those Guide pages to a minimum (maybe even merge them to one), with the main purpose of showcasing Codeception's syntax and overall approach.
I guess this was the original idea of the pages in the "use cases" menu (which I've already started to empty out) - so these could finally be removed then.
2. Move all detail information (e.g. installation) to the module pages.

**Comment by DavertMik** [comment]
No, that was not the original plan.

I agree that some pages need reorganization but I don't want to cut out and merge guides, quite contrary I would like to add more pages and extend current guides with more practical examples. 

Module pages can't be extended truth as they are written inside source code.
